include/session.php

```diff
@@ -30,7 +30,7 @@ function ref_session_read($id) {
 
 	$r = q("SELECT `data` FROM `session` WHERE `sid`= '%s'", dbesc($id));
 
-	if (dbm::is_result($r)) {
+	if(dbm::is_result($r)) {
```

Comment:
@annando okay, will add a space in all occurrances with `if(dbm::is_result($bla)`. Still I can find more in the code (only a side-note).